Mathlib/RingTheory/OreLocalization/OreSet.lean

```diff
@@ -21,6 +21,23 @@ This defines left Ore sets on arbitrary monoids.
 
 -/
 
+namespace AddOreLocalization
+
+/-- A submonoid `S` of an additive monoid `R` is (left) Ore if common summands on the right can be
```

Comment:
Following the changes in `Basic` I'd change this "common factors" (and related things) to the translations.

Reply:
Sure. I just pushed [a commit](https://github.com/leanprover-community/mathlib4/pull/13559/commits/932f692aa7ff4d12fc2020b0daef5a6039ab63fb) fixing them. Please double check that the wording is good.